src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/hooks.test.js

```diff
@@ -263,6 +263,10 @@ describe('Problem settings hooks', () => {
       {
         correct: true,
         id: 'b',
+      },
+      {
+        correct: false,
+        id: 'c',
```

Comment:
If you add a test that fails when multiple answers stay selected in the dropdown but succeeds after your fix, this will prevent the same bug from reappearing due to future changes.

Reply:
The bug was in that the correctAnswerCount was negative so multiple answers stay selected because the count was never greater than one. I added a test that makes sure only answers with correct true are called because the underlying issue was updateAnswer being called extraneous times.